apps/web/app/components/FileUploadInput/ImageUploadInput.tsx

```diff
@@ -17,9 +17,11 @@ const ImageUploadInput = ({ field, handleImageUpload, isUploading }: ImageUpload
           <img src={field.value} alt="Uploaded" className="w-full h-full object-cover" />
         )}
         <div
-          className={cn("absolute inset-0 flex flex-col items-center justify-center text-center", {
-            "text-white": field.value,
-          })}
+          className={cn(
+            `absolute inset-0 flex flex-col items-center justify-center text-center ${
+              field.value ? "text-white" : ""
+            }`,
+          )}
```

Comment:
**question:** Could something have gone wrong here, perhaps during the merge process?